src/HDInsight/HDInsight/ManagementCommands/SetAzureHDInsightGatewayCredentialCommand.cs

```diff
@@ -0,0 +1,151 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Hyak.Common;
+using Microsoft.Azure.Commands.HDInsight.Commands;
+using Microsoft.Azure.Commands.HDInsight.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.HDInsight.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Common;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.HDInsight
+{
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightGatewayCredential", DefaultParameterSetName = SetByNameParameterSet, SupportsShouldProcess = true), OutputType(typeof(HttpConnectivitySettings))]
+    public class SetAzureHDInsightGatewayCredentialCommand : HDInsightCmdletBase
+    {
+        private const string SetByNameParameterSet = "SetByNameParameterSet";
+        private const string SetByResourceIdParameterSet = "SetByResourceIdParameterSet";
+        private const string SetByInputObjectParameterSet = "SetByInputObjectParameterSet";
+
+        #region Input Parameter Definitions
+
+        [Alias("ClusterName")]
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
Remove ValueFromPipelineByProeprtyName property from this attribute - it does not apply

Reply:
> Remove ValueFromPipelineByProeprtyName property from this attribute - it does not apply

Fixed